### PR TITLE
GR: implement Bonded Auctioneer

### PR DESCRIPTION
**Pull request by strib**

### CODE REVIEW

File: server/game/cards/07-GR/BondedAuctioneer.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            }
+        });
+
+        this.scrap({
+            effect: "return {0} to its owner's hand",
+            effectArgs: (context) => [context.target],
+            target: {
+                cardType: 'artifact',
+                location: 'play area',
+                gameAction: [ability.actions.returnToHand()]
```
Comment (romanogit) [inline server/game/cards/07-GR/BondedAuctioneer.js:29]: I believe you can avoid the array and the `effect` property on line 24.

File: server/game/cards/07-GR/BondedAuctioneer.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    //
+    // Scrap: Return an artifact to its owner’s hand.
+    setupCardAbilities(ability) {
+        this.reap({
+            effect: 'destroy {0} and have {1} gain 1 amber',
+            effectArgs: (context) => [context.target.controller],
+            target: {
+                cardType: 'artifact',
+                location: 'play area',
+                gameAction: [ability.actions.destroy()]
```
Comment (romanogit) [inline server/game/cards/07-GR/BondedAuctioneer.js:14]: here, you can avoid the array as well.

File: server/game/cards/07-GR/BondedAuctioneer.js (hunk 1 above)
Comment (strib) [reply server/game/cards/07-GR/BondedAuctioneer.js]: Thanks, done.

File: server/game/cards/07-GR/BondedAuctioneer.js (hunk 2 above)
Comment (strib) [reply server/game/cards/07-GR/BondedAuctioneer.js]: Done.

**Pull request merged**